x/erc20/genesis_test.go

```diff
@@ -39,7 +104,7 @@ func (suite *GenesisTestSuite) SetupTest() {
 	suite.app = app.Setup(false, feemarkettypes.DefaultGenesisState())
 	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{
 		Height:          1,
-		ChainID:         "canto_9000-1",
+		ChainID:         "test-chainid",
```

Comment:
The chainID of Ethermint chains should be EIP155 Ethereum-compatible [Ref](https://github.com/Canto-Network/ethermint/blob/db8920e278dbf6ed03aa63212f8bdd3f83b6e68d/types/chain_id.go#L1-L50)

if you keep `canto_9000-1`,  we can refactor at a time

Reply:
Please let me know if it was an intentional correction or if you had any other intentions